yt/visualization/volume_rendering/scene.py

```diff
@@ -367,7 +367,7 @@ def save_annotated(
         dpi: int = 100,
         sigma_clip: Optional[float] = None,
         render: bool = True,
-        tf_rect: Optional[list] = None,
+        tf_rect: Optional[List[float]] = None,
```

Comment:
sorry, this is my first experience with typing.

Reply:
no worries. Admittedly typing is a hassle. In Python 3.9, the builtin name `list` can be used instead of `typing.List`, which makes things easier, but we still support Python 3.7 so...

Reply:
this should be fixed now